packages/api/src/api/videos.ts

```diff
@@ -0,0 +1,47 @@
+import {Router} from 'express';
+import { Video } from '../entities/Video';
+import logger from '../logger';
+
+export const videos = Router();
+
+// Get all videos
+videos.get('', async (req, res) => {
+    try {
+        const videosRepository = req.entityManager.getRepository(Video);
+        const videosList = await videosRepository.findAll();
+
+        res.status(200).send(videosList);
+    } catch (error) {
+        logger.error('There was an issue geting all videos: ', error);
+        res.status(500).send('There was an issue geting all videos');
+    };
+});
+
+// Get video by id
+videos.get('/:videoId', async (req, res) => {
+    const { videoId } = req.params;
+
+    try {   
+        // Check if videoId is in the expected format
+        if (isNaN(+videoId)) {
+            throw new TypeError();
+        }  
```

Comment:
IMO I consider the `plus` thing to be kind of a hack and confusing to newer devs so I'd like to avoid that whenever possible.

Also instead of throwing an error you can just respond here and avoid the if statement later.

```suggestion
        if (isNaN(Number(videoId))) {
            res.status(400).send('The id must be a number');
            return;
        }  
```

Reply:
Updated.